action.yml

```diff
@@ -315,6 +334,12 @@ runs:
         PATH=$PATH:$(pwd)
         cd $GITHUB_WORKSPACE
         digger
+
+    - uses: actions/cache/save@v4
+      name: cache-save
+      with:
+        path: ${{ github.workspace }}/cache
+        key: ${{ runner.os }}-${{ hashFiles('**/cache') }}
```

Comment:
should we also have:

```
 if: ${{ inputs.cache-dependencies == 'true' }}
```

here?

Reply:
Yep realized that when I was reviewing in a break between, so added :)